radioactive-decay/exercises/q_radioactive_decay.yml

```diff
@@ -0,0 +1,5 @@
+uid: q-radioactive decay 
+title: Radioactive Decay
+type: Derivation
+options:
+  text: Derive the Poisson distribution from the binomial distribution.
```

Comment:
This was also for practice. The complete problem has them compute the 1st and 2nd moments of the distribution and figure out what it means. (They're equal!). You can see the problem in the SIIP repository.